### PR TITLE
handle protected access

### PR DESCRIPTION
**Pull request by elharo**
@suztomo

### CODE REVIEW

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	
+	private ClassFile file;
+	private Symbol symbol;
+	
+	@Before
+	public void setUp() {
+      Path path = Paths.get("/usr/tmp");
+      ClassPathEntry entry = new ClassPathEntry(path);
+      file = new ClassFile(entry, "foo");
+      symbol = new Symbol("") {
```
Comment (suztomo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:36]: `ClassSymbol("Foo")`? This is shorter.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblem.java [hunk 2]
```diff
@@ -42,15 +42,18 @@ public final String toString() {
         message.append(" is public");
         break;
       case PRIVATE:
-        message.append(" is private");
-        break;
+          message.append(" is private");
```
[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblem.java:45]: done

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
```
[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:21]: done

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class InaccessibleClassProblemTest {
+	
+	private ClassFile file;
```
[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:28]: done

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java (hunk 1 above)
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:36]: done

**Review by elharo — COMMENTED** [review]
PTAL

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (suztomo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:27]: I still see the indentation is inaccurate in this file (screenshot below). Can you check any tab character or unexpected white space characters?

Screen Shot 2020-11-16 at 10 32 47

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+	private Symbol symbol;
+	
+	@Before
+	public void setUp() {
+      Path path = Paths.get("/usr/tmp");
+      ClassPathEntry entry = new ClassPathEntry(path);
```
Comment (suztomo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java:33]: Strange indentation

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/InaccessibleClassProblemTest.java]: done

**Review by suztomo — APPROVED** [review]

**Pull request merged**